core_gems.rb

```diff
@@ -1,13 +1,16 @@
 dir 'core_gems'
 download "json", "2.1.0"
-download "msgpack", "1.2.2"
-download "cool.io", "1.5.3"
+download "msgpack", "1.2.9"
+download "cool.io", "1.5.4"
 download "http_parser.rb", "0.6.0"
 download "yajl-ruby", "1.3.1"
 download "sigdump", "0.2.4"
 download "thread_safe", "0.3.5"
 download "oj", "3.3.10"
 download "tzinfo", "1.2.2"
 download "tzinfo-data", "1.2016.5"
-fetch "google-protobuf", "3.7.1"
-fetch "grpc", "1.20.0"
+unless windows?
+  # Build dependencies for 'google-fluentd' that aren't needed on Windows.
```

Comment:
More precisely, they are needed on Windows, but are installed through another mechanism.

Reply:
Thanks for the clarification.

Reply:
I was proposing a change for a misleading comment.

Reply:
I was proposing a change for a misleading comment. How about:
```
  # Build dependencies for 'google-fluentd' that don't need to be fetched explicitly on Windows.
```
?
Or, since this whole file is build dependencies for google-fluentd:
```
  # Gems that don't need to be fetched explicitly on Windows.
```
?